map/map_loader/src/lanelet2_map_loader/lanelet2_map_loader_node.cpp

```diff
@@ -105,13 +108,13 @@ lanelet::LaneletMapPtr Lanelet2MapLoaderNode::load_map(
   if (projector_info.projector_type != tier4_map_msgs::msg::MapProjectorInfo::LOCAL) {
     std::unique_ptr<lanelet::Projector> projector =
       geography_utils::get_lanelet2_projector(projector_info);
-    const lanelet::LaneletMapPtr map = lanelet::load(lanelet2_filename, *projector, &errors);
+    lanelet::LaneletMapPtr map = lanelet::load(lanelet2_filename, *projector, &errors);
```

Comment:
Why was the "const" removed here?

Reply:
It is came from [performance-no-automatic-move](https://releases.llvm.org/14.0.0/tools/clang/tools/extra/docs/clang-tidy/checks/performance-no-automatic-move.html).

Seems `const` keyword prevent the compiler to use move automatically.
In some cases, this may lead to a large memory copy issue, but considering the type `lanelet::LaneletMapPtr`, this may not be expensive as mentioned.

It was marked as a `warning`, so we can revert it if using `const` is more readable.

Reply:
I see. I understand.
Thanks!